Source/Core/Core/MemoryWatcher.cpp

```diff
@@ -122,8 +138,16 @@ void MemoryWatcher::Step()
       // Update the value
       current_value = new_value;
       std::string message = ComposeMessage(address, new_value);
+#ifdef _WIN32
+      unsigned long m_written;
```

Comment:
This is not a class member variable anymore, it should not have an `m_` prefix.

Reply:
Whoops! Fixed.